templates/vue/src/components/tyde/activities/CircleOfSupport/onboarding/ObFinishView.vue

```diff
@@ -32,16 +32,24 @@
           </span>
         </b-col>
         <b-col v-else class="ob-connection-box"></b-col>
-        <b-col align-self="center">
-          <b-button
-            pill
-            variant="secondary"
-            class="secondary"
-            @click="$emit('ob-finish')"
-          >
-            Continue &#8594;
-          </b-button>
-        </b-col>
+        <b-button
+          v-if="circleViewEnabled"
+          pill
+          variant="secondary"
+          class="secondary mx-2"
+          @click="$emit('continue')"
+        >
+          Continue &#8594;
+        </b-button>
+        <b-button
+          v-else
+          pill
+          variant="secondary"
+          class="secondary mx-2"
+          @click="$emit('done')"
+        >
+          Done
+        </b-button>
```

Comment:
Since the siblings of these elements are `b-col`, it doesn't make sense that this is not wrapped in a `b-col`. It's incorrect to mix `b-col` with other elements like this.